crates/rover-client/src/operations/subgraph/introspect/types.rs

```diff
@@ -1,14 +1,9 @@
 use crate::operations::subgraph::introspect::runner::subgraph_introspect_query;
 
 pub(crate) type QueryVariables = subgraph_introspect_query::Variables;
-pub(crate) type QueryResponseData = subgraph_introspect_query::ResponseData;
-
-use std::collections::HashMap;
 
 #[derive(Debug, Clone, Eq, PartialEq)]
-pub struct SubgraphIntrospectInput {
-    pub headers: HashMap<String, String>,
-}
+pub struct SubgraphIntrospectInput {}
```

Comment:
Could this just be a unit struct?
```suggestion
pub struct SubgraphIntrospectInput;
```

Reply:
This could be, but usually I defer to using an empty struct unless it's specifically a marker / empty by design. In this case, we're just not expecting anything as input, but could be expanded to require something later.
All said and done, they're exactly the same in practice it's just a personal preference here.